website/docs/getting-started/for-engineers.md

```diff
@@ -1,9 +1,110 @@
 ---
 title: For engineers
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
 order: 102
 ---
 
-## Lorem ipsum dolor
+The Design Systems team maintains design tokens, icons, and components in a [monorepo](https://github.com/hashicorp/design-system) and publishes them regularly to npm.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam diam mi etiam mollis tortor vestibulum. Felis, arcu amet, nibh eget eget etiam orci aliquet. Varius facilisis magna faucibus commodo iaculis faucibus aliquet.
\ No newline at end of file
+## Components
+
+We provide components as an [Ember](https://emberjs.com/) addon with associated styles written in [Sass](https://sass-lang.com/). By installing the components, you also have access to the design tokens and CSS helper classes.
+
+### Install package
+
+```bash
+yarn add @hashicorp/design-system-components
+```
+
+### Import styles
+
+You need to install and configure Sass to preprocess styles, handle source maps, and include paths in your application.
+
+```bash
+ember install ember-cli-sass
+```
+
+Make sure the styles are being preprocessed by using the `scss` extension (for example, you may need to change `app/styles/app.css` to `app/styles/app.scss`).
+
+Add the following configuration in `ember-cli-build.js` to set the number of decimal places and enable access to the design system tokens:
+
+```js
+sassOptions: {
+  precision: 4,
+  includePaths: [
+    './node_modules/@hashicorp/design-system-tokens/dist/products/css',
+  ],
+},
+```
+
+Add the following line to the main Sass file in your application (for example, in `app.scss`):
+
+```scss
+@import '@hashicorp/design-system-components';
```

Comment:
Super nit: We use double quotes within Scss code in HDS styles. (I know consumers may have different standards though.)

Reply:
Happy to update to double 👍

Reply:
Thanks!